_pages/01-applicants.md

```diff
@@ -112,4 +112,14 @@ please mention that in the application.
 
 Yes. While we understand that not all topics may be 100% relevant to you we want to maximize engagement. Note that we post the lessons and videos online later, so if you are interested in only a portion of the OceanHackWeek consider that an option rather than taking someone's spot.
 
----
+**Q: I did not get accept, what now?**
```

Comment:
Change "accept" to "accepted"

Reply:
I guess I did not push the latest changes. The question should be the same as the issue title.